spec/concepts/api/v1/lib/service/email_token_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+RSpec.describe Api::V1::Lib::Service::EmailToken do
```

Comment:
Where checking for defined constants expectations?

Reply:
Added.